ChangeLog

```diff
@@ -1,7 +1,60 @@
 ==============================================================
 
+Tag name: noresm3_0_alpha06a
+Date: 17-06-2025
+One-line Summary: Update BLOM/iHAMOCC
+
+Details:
+ - Update BLOM/iHAMOCC to tag dev1.9.0.28:
+   - New iHAMOCC default configuration and parameter settings
+   - Define working precision for iHAMOCC
```

Comment:
@TomasTorsvik , in general looks good to me. Maybe one could add here the info that POM remin in case of DOM classes now also follows a T- and O2-dependent remineralization?